src/Materials/Textures/sampler.ts

```diff
@@ -0,0 +1,152 @@
+import { Constants } from "../../Engines/constants";
+import { Nullable } from "../../types";
+
+/**
+ * Class used to store a sampler data
+ */
+export class Sampler {

```

Comment:
As this will be exported as BABYLON.Sampler, could we call it TextureSampler (for example) to not have an abstract name in the main namespace?

Reply:
No problem but waiting for confirmation from @sebavan and/or @deltakosh as this will impact some external assets (PGs / Docs), so I prefer doing it a single time :)

Reply:
I guess TextureSampler is more along the Babylon.js Linguo, I am up for the change

Reply:
Done